metricbeat/module/kafka/broker/_meta/fields.yml

```diff
@@ -0,0 +1,50 @@
+- name: broker
+  type: group
+  description: Broker metrics from Kafka Broker JMX
+  release: beta
+  fields:
+    - name: mbean
```

Comment:
mbeans should be always in the same field at the root level (something like `jmx.mbean`).

This is probably something to change in the Jolokia module, so we can keep this here by now.